js/libs/keycloak-js/assembly.xml

```diff
@@ -0,0 +1,16 @@
+<assembly xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.3 http://maven.apache.org/xsd/assembly-1.1.3.xsd">
+    <id>dist</id>
+    <formats>
+        <format>zip</format>
+    </formats>
+    <includeBaseDirectory>false</includeBaseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>${project.basedir}/target</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>keycloak-*.tgz</include>
+            </includes>
+        </fileSet>
+    </fileSets>
+</assembly>
```

Comment:
This is not needed, we should rather just attach `keycloak-js-999.0.0-SNAPSHOT.tgz` as it is already created by `npm pack`.

The format of the tgz file is explicit to support installation with `npm` and other package managers. For example with the new format one can do `npm install keycloak-js-999.0.0-SNAPSHOT.tgz`.